### ISSUE TITLE
writing a costum loss function for differentiable counting 

### ISSUE THREAD

**Issue by Bajo1994** [pytorch/pytorch#52685]
Imagine we are asked to write a model (for instance combination of a couple of FC layers) to generate numbers from random values. The goal is to generate 30% of the output above (not equal) 10.0 and 70% of the outputs below (not equal) 10.0. I am aware that counting and comparison are not differentiable in PyTorch. How can I write a costume loss function to train this model?

**Comment by ailzhang** [comment]
Please redirect all questions to pytorch forum. Thanks!
